### DISCUSSION TITLE
H1 title vs YAML title

### DISCUSSION THREAD

**Discussion by rgaiacs** [quarto-dev/quarto-cli#3204] [Q&A] [answered]
What is the difference of using

```
# Quarto is awesome
```

instead of

```
---
title: Quarto is awesome
---
```

From a quick test, I need to use H1 for cross reference in books. For example,

```
# Quarto is awesome {#sec-quarto}

See @sec-quarto
```

will work when

```
---
title: Quarto is awesome {#sec-quarto}
---

See @sec-quarto
```

will fail to create the cross reference.

![Screenshot 2022-11-07 at 14-05-32 Quarto is awesome {#sec-quarto}](https://user-images.githubusercontent.com/1506457/200237447-59e8096e-d123-4bda-b810-83bd50d29d2b.png)

**Comment by mcanouil** [comment, marked as answer]
`title` is used in `header` HTML tag with `title-block-header` id in a `div` with `.quarto-title` class.

`# Title` is regular h1 title embedded in `section` HTML tag with `title` id.


You can see this using your browser developper mode.

```
---
title: Title
format: html
---

# Title
```

```html
<main class="content" id="quarto-document-content">
  <header id="title-block-header" class="quarto-title-block default">
    <div class="quarto-title">
      <h1 class="title">Title</h1>
    </div>
    <div class="quarto-title-meta">
    </div>
  </header>
  <section id="title" class="level1">
    <h1>Title</h1>
  </section>
</main>
```
